alvr/server/src/connection.rs

```diff
@@ -657,8 +655,9 @@ fn connection_pipeline(
     )?;
 
     let mut video_sender = stream_socket.request_stream(VIDEO);
-    let game_audio_sender = stream_socket.request_stream(AUDIO);
-    let mut microphone_receiver = stream_socket.subscribe_to_stream(AUDIO, MAX_UNREAD_PACKETS);
+    let game_audio_sender: alvr_sockets::StreamSender<()> = stream_socket.request_stream(AUDIO);
+    let mut microphone_receiver: alvr_sockets::StreamReceiver<()> =
```

Comment:
Import `StreamSender` with `use`

Reply:
Both. you seem to need to specify the variable type for both

Reply:
No i meant which type i should use for StreamSender/StreamReceiver, it's a generic

Reply:
there was some discussion about this with @The-personified-devil in discord

Reply:
You import the generic type and then do specialization where you use the type.

Reply:
So `use alvr_sockets::StreamSender;` and then `let game_audio_sender: StreamSender<()>`